extensions/ui/iconset/org.eclipse.smarthome.ui.iconset.classic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
```

Comment:
> Why do you need an own bundle for the ClassicUI Icon set?

Because it is not the ClassicUI icon set, but the Classic icon set. It is used by the Android app, the iOS app, the Basic UI and soon also the Paper UI as well :-)

> And does it really contain 4 Icons only?

This is the status quo, see https://github.com/eclipse/smarthome/tree/master/distribution/smarthome/runtime/icons.
But once this PR is merged, I plan to have the new openHAB 2 icon set contributed to ESH, see https://github.com/openhab/openhab2/pull/291#issuecomment-119709130.